bpf/Makefile.bpf

```diff
@@ -77,9 +77,15 @@ checkpatch:
 		-e GITHUB_REF=$(GITHUB_REF) -e GITHUB_REPOSITORY=$(GITHUB_REPOSITORY) \
 		$(CHECKPATCH_IMAGE_AND_ENTRY) $(CHECKPATCH_ARGS)
 
+# Note: --macro-file-builtins and --iso-file can be removed, here and in the
+# messages in the .cocci files, next time we upgrade coccinelle. The issue was
+# fixed in https://gitlab.inria.fr/coccinelle/coccinelle/-/commit/540888ff426e.
+COCCI_PATHS ?= --macro-file-builtins /usr/local/lib/coccinelle/standard.h --iso-file /usr/local/lib/coccinelle/standard.iso
 coccicheck:
 	$(QUIET) $(foreach TARGET,$(shell find $(ROOT_DIR)/contrib/coccinelle/ -name '*.cocci'), \
-		spatch --no-show-diff --sp-file ${TARGET} --include-headers --very-quiet --dir . ; )
+		spatch --no-show-diff --sp-file ${TARGET} --include-headers --very-quiet \
```

Comment:
nit: avoid mixing `$()` and `${}`
```suggestion
		spatch --no-show-diff --sp-file $(TARGET) --include-headers --very-quiet \
```

Reply:
Only nits and a reject review. You're a harsh reviewer! :smile_cat:

Reply:
:)

I don't know, “Changes requested” means we expect the PR to be updated, I don't find it harsh. Makes reviewer's intention clear, at least.

Then of course I'd adjust depending on who is the author, it's true the “rejection” aspect might discourage newer contributors.

Reply:
@pchaigno for context at the time I reviewed CI didn't pass, if it did I wouldn't have requested changes.